src/main.py

```diff
@@ -85,14 +85,33 @@ def mensa(update, context):
                 markdown_highlight_char = "*"
 
         img_url = elem["image"].lstrip("/") # For some reason, image URLs are prefixed with 2 leading slashes, but no protocol, remove them
-        print(img_url)
         # Do not send placeholder images
         if img_url.endswith("studentenwerk-dresden-lieber-mensen-gehen.jpg"):
             context.bot.send_message(chat_id=update.message.chat_id, text=markdown_highlight_char + elem["name"] + markdown_highlight_char, parse_mode="Markdown")
         else:
             context.bot.send_photo(chat_id=update.message.chat_id, photo=img_url, caption=markdown_highlight_char + elem["name"] + markdown_highlight_char, parse_mode="Markdown")
 
 
+def alte_mensa(update, context):
```

Comment:
It's not really clear where these numbers (4, 35 etc) come from. Maybe some more explanation (comments etc) or even a new variable for each number is more meaningful?

Reply:
I added a comment right on top of the mensa function.

I'd prefer not to introduce new variables for this purpose. IMHO the process is pretty easily understandable as it is right now.
No need to add more complexity.